README.md

```diff
@@ -1 +1,140 @@
-# Budget-app
\ No newline at end of file
+# Budget App
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Live Demo](#live-demo)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Budget app] <a name="about-project"></a>
+
+The **Budget App** is a Ruby on Rails mobile web application where you can manage your budget: you have a list of transactions associated with a category so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+Ruby, Ruby on Rails, Postgresql, Visual Studio Code.
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby on Rails</summary>
+</details>
+
+<details>
+  <summary>PostgreSQL</summary>
+</details>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- ✅ Click [Live Demo Link](https://mic-0dbi.onrender.com)
+- ✅ Click [here](https://www.loom.com/share/da6ea82c6ee54a6e986693582c8bce38?sid=9540801e-eb98-443d-8e0b-52df6a6b8327) to see the video presentation.
```

Comment:
- [x] Good Job for documenting your work professionally and provide the link to video on the README file  But as stated in the  project requirement the video should mention the following points clearly.
- The video contains a demo of the project, highlighting all the project features(So kindly include the highlight of Sign up, Login page , On the Homes page should highlight how to add a category but your video only highlight about the how to "Add a new transaction" page on the existing category)
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making. So kindly record an another video  by including the above mention point .

- [x] Good Job for deploys the application using Render, and provide A link to the deployed project is in the README file but kindly make sure that project is deployed properly, the home and detail page works as expected.(I see that you mention it's related to ruby  version you used for the project 3.3.0 is not compatible with the render version render site use version ruby 2.2 but i think these issue can be resolved because for the same project budget app I used ruby version '3.2.2' for project and deploy it on the render you can check the application link [here](https://fast-budget-expense-manager-application.onrender.com/)  ).

Reply:
hey @Abenezer-Tilahun for project deploy I how is used  ruby version '3.3.0' for my  project and just change to and when i try to deploy the log of render told me the ruby version should be 2.2 to solve the issue of deploy and it worked just to deploy when i change the ruby version to 2.2 via GitHub browser that's i told you this issue is out of my hand for now ?